_docs_use-cases/use-case-transport-create-qr-code-for-file.md

```diff
@@ -19,7 +19,7 @@ properties:
   - tech category: Files
   - status: DONE
   - documentation status: DONE
-  - comments: Accept: image/png
+  - comments: image/png for Accept
```

Comment:
I also mentioned in another PR `Accept: image/png` makes clear that `image/png` stands in the `Accept` header, `image/png for Accept` is very confusing.